oak/server/oak_runtime.cc

```diff
@@ -108,6 +110,39 @@ grpc::Status OakRuntime::Initialize(const ApplicationConfiguration& config) {
   return grpc::Status::OK;
 }
 
+std::string OakRuntime::NextNodeName(const std::string& contents) {
+  absl::MutexLock lock(&mu_);
+  int index = next_index_[contents]++;
+  return absl::StrCat(contents, "-", index);
+}
+
+bool OakRuntime::CreateWasmNode(const std::string& contents, std::unique_ptr<ChannelHalf> half,
+                                std::string* node_name) {
+  if (wasm_contents_.count(contents) != 1) {
+    LOG(ERROR) << "failed to find Wasm contents with name " << contents;
```

Comment:
"Wasm contents" seems a bit vague, how about "Wasm module" or "Wasm bytecode"?

Reply:
Done (although it will get re-jigged in the later PR).